guides/common/modules/proc_creating_an_application_definition.adoc

```diff
@@ -0,0 +1,30 @@
+[id="{context}_creating_an_application_definition"]
+= Creating an Application Definition
+
+Use this procedure to create xref:{context}_application_definitions[application definitions] to use as part of your application centric deployment.
+
+.Prerequisites
+You need existing host groups in order to use application centric deployment.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Applications > App Definitions*.
+. Click *New Application Definition*.
+. In the *Name* field, enter the name of the new application definition.
+. In the *Description* field, enter an arbitrary description.
+. From the *Ansible Playbook* list, select the name of an Ansible playbook.
+. Click the plus icon to add new sets of hosts.
+.. In the *Name* field, enter the name of the new set of hosts.
+.. In the *Description* field, enter an arbitrary description.
+.. From the *Hostgroup* menu, select an existing host group.
+.. From the *Ansible Group* menu, select existing Ansible group variables from the selected Ansible playbook.
+.. In the *min count* field, enter the minimum number of hosts providing this service.
+If it is unset, users can choose to not deploy a host providing this service at all.
+.. In the *max count* field, enter the maximum number of hosts providing this service.
+If it is unset, users can choose to deploy an unlimited number of hosts providing this service.
+.. In the *Actions* column,
+... click the *edit* icon to edit an entry;
+... click the *settings* icon to add or edit existing {ProjectName} parameters.
```

Comment:
```suggestion
... Click the *settings* icon to add or edit existing {ProjectName} parameters.
```

Reply:
Are all of these icons really lower case in the web UI?

Reply:
It doesn't say `settings` anywhere, it's an icon. Hovering it shows `change paramters`.